scripts/components/Interactions/OpenContent.js

```diff
@@ -485,3 +494,9 @@ OpenContent.SIZE_MIN = 64;
 
 /** @constant {number} SIZE_MAX Maximum size for content. */
 OpenContent.SIZE_MAX = 512;
+
+/** @constant {number} SIZE_MAX_360 Maximum size for 360 content */
+OpenContent.SIZE_MAX_360 = 1024;
```

Comment:
@janlindso Do you have any preference for the max size? We didn't specify it clearly. And do you need separate values for width and height?

Reply:
This is good!